Dockerfile

```
@@ -24,12 +25,7 @@ COPY --chown=build:build . /src
 RUN : \
     && cd src \
     && if [ -f configure ]; then echo "Run git clean -fX" >&2; exit 1; fi \
-    && scripts/compile 5.6 /dst \
-    && scripts/compile 7.0 /dst \
-    && scripts/compile 7.1 /dst \
-    && scripts/compile 7.2 /dst \
-    && scripts/compile 7.3 /dst \
-    && scripts/compile 7.4 /dst \
+    && scripts/compile --output-dir /dst 5.6 7.0 7.1 7.3 7.4 8.0 \
```

```suggestion
    && scripts/compile --output-dir /dst 5.6 7.0 7.1 7.2 7.3 7.4 8.0 \
```

Unless 7.2 is being dropped specifically?

---

Typing is hard